contents/navigation/paginations/simple-button/index.tsx

```diff
@@ -1,9 +1,16 @@
-import { Box, Button, Pagination, Text, ui } from "@yamada-ui/react"
+import { Box, Pagination, Text, ui } from "@yamada-ui/react"
 import { useState, type FC } from "react"
 
 const SimpleButton: FC = () => {
   const [page, onChange] = useState<number>(1)
 
+  const buttonStyles = {
```

Comment:
I want you to put a mold on it.

```suggestion
  const buttonStyles:HTMLUIProps<"button"> = {
```

Reply:
Sure!!